docs/projects/amt/reporting-standard/latest.md

```diff
@@ -73,10 +73,16 @@ A `system_card` contains the following information.
     4. `author` (OPTIONAL, string). Name of person that initiated the transformations.
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instruments` (OPTIONAL, list). List of instruments from
+the [instrument register](https://github.com/MinBZK/instrument-registry) that must be executed to fill this system card.
+There can be multiple instruments. For each instrument the following fields are present.
+    1. `urn` (OPTIONAL, string). Uniform Resource Names of the instrument.
```

Comment:
@ChristopherSpelt, when an instrument object is added, shall we make the `urn` required?